pages/content/amp-dev/documentation/components/reference/amp-3d-gltf@ar.md

```diff
@@ -0,0 +1,121 @@
+---
+$category@: media
+formats:
+  - websites
+teaser:
+  text: Displays GL Transmission Format (gITF) 3D models.
+---
+
+<!--
```

Comment:
wow, so strange to see the license header translated ;) we can probably change those back to English.